src/test/java/com/hubspot/jinjava/util/ObjectTruthValueTest.java

```diff
@@ -15,6 +14,46 @@ public void itEvaluatesObjectsWithObjectTruthValue() {
       .isFalse();
   }
 
+  @Test
+  public void itEvaluatesIntegers() {
```

Comment:
might be worth a utility function here that takes a and b and does the assertions so you're not duplicating the code for every type.